plugins/node/opentelemetry-instrumentation-knex/src/instrumentation.ts

```diff
@@ -45,8 +45,8 @@ const DEFAULT_CONFIG: types.KnexInstrumentationConfig = {
 export class KnexInstrumentation extends InstrumentationBase {
   constructor(config: types.KnexInstrumentationConfig = {}) {
     super(
-      `@opentelemetry/instrumentation-${constants.MODULE_NAME}`,
-      VERSION,
+      PACKAGE_NAME,
```

Comment:
same as router - As part of this change I think we could remove the `MODULE_NAME` constant from the `constants.ts` file. Then we can also rename in the `init()` there is another `constants.MODULE_NAME`.